File: quality/tests/fixtures/ruby/sample.rb
# Conformance fixture: known escapes and one function of known complexity.
# rubocop:disable Style/Foo
def read
  skip "later"
end

def branchy(n)
  return 1 if n == 1
  return 2 if n == 2
  return 3 if n == 3
  return 4 if n == 4
  0
end

File: quality/tests/fixtures/go/sample.go
// Conformance fixture: known escapes and one function of known complexity.
package sample

import "testing"

//nolint:errcheck
func read() {}

func TestSkipped(t *testing.T) { t.Skip("later") }

func branchy(n int) int {
	if n == 1 {
		return 1
	}
	if n == 2 {
		return 2
	}
	if n == 3 {
		return 3
	}
	if n == 4 {
		return 4
	}
	return 0
}
